### PR TITLE
Add support for reading PFM from Cys2His2 Zinc Finger Proteins PWM Predictor.

### PR DESCRIPTION
**Pull request by ghuls**
- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``pre-commit``
locally, and understand that continuous integration checks will be used to
confirm the Biopython unit tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

Add support for reading PFM from Cys2His2 Zinc Finger Proteins PWM Predictor (http://zf.princeton.edu/logoMain.php).

### CODE REVIEW

File: Tests/motifs/fourrows.pfm [hunk 1]
```diff
@@ -47,3 +47,12 @@ T  [ 2 19 11 50 29 47 22 81  1  6 ]
...
 2 19 11 50 29 47 22 81  1  6
+
+# Cys2His2 Zinc Finger Proteins PWM Predictor
+base       1       2       3       4       5       6       7       8       9 
+   a   0.116   0.974   0.444   0.116   0.974   0.444   0.667   0.939   0.068 
+   c   0.718   0.006   0.214   0.718   0.006   0.214   0.143   0.006   0.107 
+   g   0.016   0.020   0.028   0.016   0.020   0.028   0.047   0.045   0.216 
+   t   0.150   0.001   0.314   0.150   0.001   0.314   0.143   0.009   0.609 
+
+Ent=   1.210   0.202   1.665   1.210   0.202   1.665   1.399   0.396   1.521
```
Comment (mdehoon) [inline Tests/motifs/fourrows.pfm:58]: What does the `Ent` line mean?

Comment (ghuls) [reply Tests/motifs/fourrows.pfm:58]: `Ent` seems to be `2-relative entropy`:

```
In [17]: motif.counts
Out[17]: 
{'G': [0.016, 0.02, 0.028, 0.016, 0.02, 0.028, 0.047, 0.045, 0.216],
 'A': [0.116, 0.974, 0.444, 0.116, 0.974, 0.444, 0.667, 0.939, 0.068],
 'T': [0.15, 0.001, 0.314, 0.15, 0.001, 0.314, 0.143, 0.009, 0.609],
 'C': [0.718, 0.006, 0.214, 0.718, 0.006, 0.214, 0.143, 0.006, 0.107]}

In [18]: 2 - motif.relative_entropy
Out[18]: 
array([1.20966654, 0.20538403, 1.66527285, 1.20966654, 0.20538403,
       1.66527285, 1.39950626, 0.39098754, 1.52201241])

In [19]: ! cat pwm.txt
base       1       2       3       4       5       6       7       8       9 
   a   0.116   0.974   0.444   0.116   0.974   0.444   0.667   0.939   0.068 
   c   0.718   0.006   0.214   0.718   0.006   0.214   0.143   0.006   0.107 
   g   0.016   0.020   0.028   0.016   0.020   0.028   0.047   0.045   0.216 
   t   0.150   0.001   0.314   0.150   0.001   0.314   0.143   0.009   0.609 

Ent=   1.210   0.202   1.665   1.210   0.202   1.665   1.399   0.396   1.521 
```

**Comment by mdehoon** [comment]
Can you update Doc/Tutorial/chapter_motifs.rst to include this format?

**Comment by mdehoon** [comment]
Finished?

**Comment by ghuls** [comment]
Yes.

**Pull request merged**